index.html

```diff
@@ -2440,8 +2445,19 @@ <h5>Conclusion</h5>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
      Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-  <!-- short summary of how the tag applies -->
+<li>
+<a data-ucr-role="tag" href="#accessibility-potential"></a>
+Web maps provided by existing reference tools
+tend to disregard accessibility best practices,
+and assistive technology users have poor to middling experiences with web maps,
+so much so that it is common for developers to hide them completely from assistive technology.
```

Comment:
The last statement is definitely from personal experience/conversations I've observed amongst a11y professionals, but I don't think it can be quantified, so I will remove it. For the 'tend to disregard' part, could we cite other sections of the report? I think lack of visible focus state could constitute as example of disregarding best practice.

Reply:
It doesn't really help that this initial "create a web map" capability is overly broad, so any conclusions would also be overly broad. But I'd rather leave the specific critiques for the specific sections where we talk about the problems with current practice.

Reply:
Maybe we could add a general statement in the prose for this section, about how more specific concerns and areas for improvement will be listed in the discussions of individual capabilities?